### PR TITLE
Fix run benchmark supervisor script

### PR DESCRIPTION
**Pull request by pierDipi**
Signed-off-by: Pierangelo Di Pilato <pierdipi@redhat.com>

### CODE REVIEW

File: bin/run_benchmark.py [hunk 1]
```diff
@@ -156,10 +169,19 @@ def is_terminated(run_id: str) -> bool:
 
 def is_failed(run_id: str) -> bool:
     print(run_id, f"Checking benchmark run success/failure")
-    info = get_run_info(run_id)
+    info = get_run_stats(run_id)
     response = json.loads(info.read())
+    stats = response["statistics"]
     pprint.pprint(response)
-    return response["errors"] is not None and len(response["errors"]) > 0
```
Comment (pierDipi) [inline bin/run_benchmark.py:162]: errors are common and expected, and they include jitter warnings, etc, so `failedSLAs` is the correct field.

Comment (matzew) [reply bin/run_benchmark.py:162]: `failedSLAs` is in the response from?

Comment (matzew) [reply bin/run_benchmark.py:162]: ah, from hyperfoil

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**